velox/exec/PartitionedOutputBufferManager.h

```diff
@@ -265,7 +284,8 @@ class PartitionedOutputBufferManager {
  private:
   // Retrieves the set of buffers for a query.
   std::shared_ptr<PartitionedOutputBuffer> getBuffer(const std::string& taskId);
-
+  std::shared_ptr<PartitionedOutputBuffer> getBufferIfExists(
```

Comment:
This is a nice method. Please, add empty line before this method and document it. Let's use this method in PartitionedOutputBufferManager::deleteResults as well.